### PR TITLE
Feature/vmo 6428/update UI based on vendor metadata UI metadata schema change

### PR DESCRIPTION
**Pull request by safydy-r**

### CODE REVIEW

File: src/store/builder/index.ts [hunk 1]
```diff
@@ -361,9 +354,10 @@ export function computeBlockUiData(block?: IBlock | null) {
   }
 }
 
-export function computeBlockVendorUiData(block?: IBlock | null) {
+export function computeBlockVendorUiMetadata(_block?: IBlock | null): IFloipUIMetadata {
```
Comment (object-required) [inline src/store/builder/index.ts:357]: I renamed the function to match the new property name `vendor_metadata.floip.ui_metadata`

**Review by object-required — APPROVED** [review]
Great job 💯 
I also conducted a minor renaming and built the project.

File: src/store/builder/index.ts [hunk 2]
```diff
@@ -336,7 +336,7 @@ export function generateConnectionLayoutKeyFor(source: IBlock, target: IBlock):
   ]
 }
 
-export function computeBlockUiData(block?: IBlock | null) {
+export function computeBlockCanvasCoordinates(block?: IBlock | null): IBlockUIMetadataCanvasCoordinates {
```
Comment (object-required) [inline src/store/builder/index.ts:339]: And also this one, to match actual return value.

**Comment by bulatgab** [comment]
LGTM, approved.

Bonus question:
`uiData` can still be found in `store/trees` and in `InteractionDesigner.legacy.js`. This is all unused/legacy, right?

Could we briefly chat about the history and the future of 'trees' in this repo, please? I'm really curious.

**Review by bulatgab — APPROVED** [review]

File: src/store/builder/index.ts (hunk 1 above)
Comment (bulatgab) [reply src/store/builder/index.ts:357]: Good naming is important. Thank you for these changes 👍

Comment (safydy-r) [reply src/store/builder/index.ts:357]: Awesome, thanks 🙏

**Comment by adeablay** [comment]
Hello @safydy I am done testing this. Approved.

**Pull request merged**